src/main/java/seedu/duke/PassengerParser.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke;
+
+import java.util.Arrays;
+
+public class PassengerParser {
+    private static PassengerList passengers = new PassengerList();
+
+    public static void parse(String[] inputWords) {
+        String command = inputWords[1];
```

Comment:
Similar to previous recommendation, "1" can be made into a constant variable to avoid magic numbers. i.e. COMMAND_INDEX